docs/reference/search/search-your-data/search-template.asciidoc

```diff
@@ -17,11 +17,14 @@ your searches without modifying your app's code.
 To create or update a search template, use the <<create-stored-script-api,create
 stored script API>>.
 
+
 The request's `source` supports the same parameters as the
 <<search-search-api-request-body,search API>>'s request body. `source` also
-supports https://mustache.github.io/[Mustache] variables, typically enclosed in
+supports https://mustache.github.io/[Mustache] variables and has a https://github.com/elastic/elasticsearch/blob/main/modules/lang-mustache/build.gradle/[dependency] on the open source project https://github.com/spullara/mustache.java[mustache.java]. 
+
+Typically https://mustache.github.io/[Mustache] variables are enclosed in
```

Comment:
Was the spirit of this issue to elaborate a bit more on some of the commonly used mustache features?

We already mention that variables are enclosed in double curly brackets. While we don't want to copy all of their documentation, would it make sense to mention that other functionality in Mustache is supported, such as iterating over arrays, conditionals, inverted conditionals, and delimeters? 

Another thing that could be worth noting (maybe?) _for 8.8_ is that there is a known issue with trailing commas in Mustache lists because it creates invalid parsing. This requires a workaround in order to use effectively. 

Perhaps a visual example of a mustache template that used variables and conditionals here would be more useful than explaining it in text. 

I like that you link to the full documentation if they'd like to know more.

Reply:
> Perhaps a visual example of a mustache template that used variables and conditionals here would be more useful than explaining it in text.

I agree an example would be great. _Show_ (rather than tell) how an Elasticsearch user may want to use mustache. Let the mustache docs provide all the details.